opencti-platform/opencti-front/src/utils/hooks/useGranted.ts

```diff
@@ -58,4 +58,9 @@ const useGranted = (capabilities: string[], matchAll = false): boolean => {
   return numberOfAvailableCapabilities > 0;
 };
 
+export const useIsAdmin = () => {
+  // only admins have access to the platform settings, so it's a good shortcut for now
+  return useGranted([SETTINGS]);
+};
```

Comment:
This is actually the recipe used in many places of the platform. I though it would be better to hide this under a dedicated hook.